namespaces/live-1.cloud-platform.service.justice.gov.uk/moving-people-safely-dev/resources/rds.tf

```diff
@@ -0,0 +1,40 @@
+variable "cluster_name" {
+}
+
+variable "cluster_state_bucket" {
+}
+
+module "dps_rds" {
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=5.0"
```

Comment:
Can you use the latest RDS version please
`https://github.com/ministryofjustice/cloud-platform-terraform-rds-instance`